### ISSUE TITLE
Microsoft.KeyVault/vaults/keys properties missing from schema published to schema.management.azure.com

### ISSUE THREAD

**Issue by MWL88** [Azure/azure-resource-manager-schemas#1558]
The KeyVault RP appears to be on-boarded to auto-generate the ARM schema, however the schema defined in this file 
https://schema.management.azure.com/schemas/2019-09-01/Microsoft.KeyVault.json is missing the following properties for resource `Microsoft.KeyVault/vaults/keys`:
- keyOps
- keySize
- kty

These properties are included in the schema file in the repo, however they are missing from the file published to `schema.management.azure.com`

**Comment by anthony-c-martin** [comment]
@apclouds - sounds like we just need to publish the latest schemas to schema.management.azure.com

**Comment by leni-msft** [comment]
@anthony-c-martin yes, confirmed the properties are there https://github.com/Azure/azure-resource-manager-schemas/blob/b83c9587a3a1e4cc77545c43f9b45eae1c34e311/schemas/2019-09-01/Microsoft.KeyVault.json#L446-L495
